pop-api/examples/balance-transfer/lib.rs

```diff
@@ -15,17 +15,17 @@ impl From<balances::Error> for ContractError {
 }
 
 #[ink::contract(env = pop_api::Environment)]
-mod pop_api_extension_demo {
+mod pop_api_balances {
```

Comment:
This couldn't be balances due to `pop_api::balances`.

Suggest we have a closer look at those names when adding examples properly working on pop api